src/content/getusermedia/resolution/js/main.js

```diff
@@ -103,7 +103,8 @@ video.onplay = function() {
 function getMedia(constraints) {
   if (stream) {
     video.src = null;
-    stream.stop();
+    stream.getAudioTracks().forEach(function(track) { track.stop(); });
```

Comment:
cc-ing @alvestrand 
one of the other things I noticed (here and in https://github.com/MicrosoftEdge/Demos/pull/68) is that we might also need a way to shim to deattach a stream from an element. Possibly attachMediaStream with a null stream.